### ISSUE TITLE
saved keras model with custom layers supports ragged inputs cannot be loaded back

### ISSUE THREAD

**Issue by tanguycdls** [tensorflow/tensorflow#38998]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): linux
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): TF2.2 rc3
- Python version: 3.6
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**Describe the current behavior**
I created a custom keras Layer that accepts ragged inputs:

```
class EmbeddingMean(tf.keras.layers.Layer):
  def __init__(self, **kwargs):
    super(EmbeddingMean, self).__init__()
    self._supports_ragged_inputs = True
  def call(self, inputs, **kwargs):
    return tf.reduce_mean(inputs, axis=1)
```
If i create a model like this:

```
feature = Input(shape=(None,), ragged=True, name='input_1', dtype=tf.int32)
embedded = Embedding(10, 3)(feature)
embedded_mean = EmbeddingMean()(embedded)
m = Model(feature, Dense(1)(embedded_mean))
```
And save the model as a keras model:
`m.save('/tmp/test')`

I cannot reload that model using 
`model_reloaded = tf.keras.models.load_model('/tmp/test')`
> ValueError: Layer embedding_mean_1 does not support RaggedTensors as input. 

If i save the model as a saved model it works correctly but i loose all the keras features.

**Describe the expected behavior**
We should be able to load back the model as expected. 

**Standalone code to reproduce the issue**
https://colab.research.google.com/gist/tanguycdls/c9b1026bbea22cca4018139f447f249c/untitled7.ipynb

**Additional Informations**
The issue seems to come from the fact that `self._supports_ragged_inputs` is not correctly saved: when we load the model the layer has the attribute set to False instead of True.

**Comment by amahendrakar** [comment]
Was able to reproduce the issue with TF v2.1, [TF v2.2.0-rc3](https://colab.research.google.com/gist/amahendrakar/883c9f51d1f7b6b8c119f115e914eb99/38998-2-2.ipynb#scrollTo=wjK4Jauu050A) and [TF-nightly](https://colab.research.google.com/gist/amahendrakar/a7d8e38707ef1af10cea71a0faa7736d/38998-tf-nightly.ipynb). Please find the attached gist. Thanks!

**Comment by markomernick** [comment]
Hey @amahendrakar-

We're going to be removing the supports_ragged_inputs attr. This change has been submitted internally and should show up in the nightlies as soon as it is sync'd.

**Comment by ymodak** [comment]
This is fixed in latest tf-nightly [commit](862a62d6b4009ab29e666e2e5c6b498c1a8b68a5), `Layer._supports_ragged_inputs` property is removed. Thanks!

**Comment by tanguycdls** [comment]
Thanks @ymodak i checked and it works ! 
thank you for the prompt answer !
